web/themes/custom/cloudy/templates/layout/cloudy-main-with-sidebar.html.twig

```diff
@@ -11,12 +11,10 @@
  */
 #}
 <div class="row">
-  <div class="col-lg-8 col-md-12">
+  <div class="cloudy-content col-lg-8 col-md-12">
```

Comment:
Did you add these classes for styling purposes or for clarity?

Reply:
Clarity. We aren't using that layout just yet, but the cloudy-content class is used for targetting TOC. If we are ever able to move to Layout Builder for the default display, this logic would need to be set up this way for header styling as well. 

As a side note, I'm not sure we'll be able to use Layout Builder on any display mode that has if/else logic in it. Layout Builder just doesn't have a means to handle fields that have a dependency on each other because each layout builder block is independent. It makes things like a checkbox to toggle the display or not display of a field kinda impossible. 🙁 

Definitely something to consider as we try to figure out how to incorporate or not incorporate layout builder in future work.

Reply:
Incidentally, I discovered these layout builder limitations when I started to pull in a content type into this layout and realized I couldn't add the logic from the event template. It was definitely a lightbulb moment.

Reply:
We are living the same life, lol. I just discovered this the other day as well, but I think I have an apporach that could work with layout builder. *fingers-crossed*